actions/daily-updates/action.yml

```diff
@@ -1,7 +1,7 @@
 name: 'daily-updates'
 description: 'Auto generates updates to the page'
 runs:
-  using: 'node12'
+  using: 'node20'
```

Comment:
Fixes "The following actions uses node12 which is deprecated and will be forced to run on node16: ./actions/daily-updates"